parm/params_grib2_tbl_new

```diff
@@ -809,6 +809,16 @@
    3 192    52   0 SBTAGR7
    3 192    53   0 SBTAGR8
    3 192    54   0 SBTAGR9
+   3 192    65   0 SBTAHI10
+   3 192    66   0 SBTAHI11
+   3 192    67   0 SBTAHI12
+   3 192    68   0 SBTAHI13
+   3 192    69   0 SBTAHI14
+   3 192    70   0 SBTAHI15
+   3 192    71   0 SBTAHI16
+   3 192    62   0 SBTAHI7
+   3 192    63   0 SBTAHI8
+   3 192    64   0 SBTAHI9
```

Comment:
Somehow, with added these new fields, the soil moisture field "SOILM" 's discipline number is changed from 2-0-22 into
2-0-3. I have worked with EMC grib2 expert on troubleshooting it. So far no smoking gun. Given the UPP develop update timeline, I would propose to backup params_grib2_tbl_new and save your changes with SBTAHI* in a new file, like params_grib2_tbl_new_dtc. We will work on unifying later.

Reply:
@WenMeng-NOAA - This solution is fine with us. We will be happy to help with the unifying process and to help identify the problem with SOILM in the future if needed.

Reply:
Thanks!

Reply:
@fossell Do you update this file on your branch? I ran the regression tests on Dell and Hera. The results looks good for me except this minor change. Thanks,!

Reply:
@WenMeng-NOAA - Sorry about that, forgot to commit that change. Should be good to go now.  The develop version of params_grib2_tbl_new has been restored and the DTC version is saved with "_dtc"

Reply:
Your update looks good. Thanks!